README.md

```diff
@@ -545,7 +555,7 @@ When pulling an image from a registry, the container runtime will try the endpoi
   endpoint = [ "https://<my-docker-hub-mirror-host>", "https://<my-docker-hub-mirror-host-2>"]
   ```
 
-  If you use a Bottlerocket variant that uses Docker as the container runtime, like `aws-ecs-1`, you should be aware that Docker only supports pull-through caches for images from Docker Hub (docker.io).  Mirrors for other registries are ignored in this case.
+  If you use a Bottlerocket variant that uses Docker as the container runtime, like `aws-ecs-1`, you should be aware that Docker only supports pull-through caches for images from Docker Hub (docker.io). Mirrors for other registries are ignored in this case.
```

Comment:
I think there might be some extra spaces at the beginning of this line.

Reply:
I believe this was intended in the original so that it is text underneath the bullet item and not a separate paragraph after the bullet list.